CancelBooking UseCase/CancelBooking.java

```diff
@@ -86,6 +88,7 @@ public boolean exists(String roomNumber, String cpr) {
 }
 // Business Layer
 // Provides methods for reservation management, including adding, canceling, and displaying reservations.
+
```

Comment:
```suggestion
the (processRefund) method could be further improved to handle cases like no-show or cancellations after check-in. also instead of having a method that handles both logic and user input/output (like cancelReservation), separate out logic from user-interfacing code.
```